src/providers/authProvider/permissions.ts

```diff
@@ -1,22 +1,27 @@
 import * as constants from '../../constants'
 type UserRoles = UserRole
```

Comment:
is there a reason why we are renaming the `UserRole` type here @TahaKhanAbdalli?